app/src/main/java/com/teamagam/gimelgimel/app/view/ImageFullscreenActivity.java

```diff
@@ -25,59 +23,39 @@
     @BindView(R.id.image_fullscreen_view)
     ZoomableDraweeView mDraweeView;
 
-    private static final String IMAGE_URI_KEY = "IMAGE_URI";
-    private static final int UI_ANIMATION_DELAY = 300;
-
     private boolean mVisible;
```

Comment:
Boolean should be prefixed with is/has/should so that if statements would read more elegant.
And its a boolean for the visibility state of the control panel.
A good name (IMO) can be mIsControlVisible
!fix